api/src/index.js

```diff
@@ -29,3 +29,61 @@ app.use("/api", apiRouter);
 app.listen(process.env.PORT, () => {
   console.log(`API listening on port ${process.env.PORT}`);
 });
+//Future-meal
+app.get('/future-meals', async (req, res) => {
+  try {
+    const now = new Date();
+    const meals = await knex('Meal').where('when', '>', now);
+    res.json(meals);
+  } catch (error) {
+    res.status(500).json({ error: error.message });
+  }
+});
+
+//past-meal
+app.get('/past-meals', async (req, res) => {
+  try {
+    const now = new Date();
+    const meals = await knex('Meal').where('when', '<', now);
+    res.json(meals);
+  } catch (error) {
+    res.status(500).json({ error: error.message });
+  }
+});
+
+//all-meal
+app.get('/all-meals',async(req,res)=>{
+  try{
+    const meals = await knex ('Meal').orderBy('id',"asc");
+    res.json(meals);
+  }catch(error){
+    res.status(500).json({error:error.message});
+  }
+ });
+
+
+ //First-meal
+ app.get('/first-meal',async(req,res)=>{
+  try{
+    const meal = await knex ('Meal').orderBy('id',"asc").first();
```

Comment:
You don't need to make implicit arguments explicit

```suggestion
    const meal = await knex ('Meal').orderBy('id').first();
```

Reply:
Hi @urbanogilson I have done the changes . Please have a look.